src/frontend/tsx/Chessboard.tsx

```diff
@@ -22,6 +22,17 @@ import { Side, CastleSide, PromotionRole, Rank, File, GameID, ActiveGameInfo } f
 import 'chessground/assets/chessground.base.css'
 import 'chessground/assets/chessground.brown.css'
 import 'chessground/assets/chessground.cburnett.css'
+import '../css/pieces.css'
+import '../css/boards.css'
```

Comment:
You shouldn't need to import local `.css` files into the `.tsx` files. You can just add them to the list of `.css` files to link in `index.html`.

I'm not sure if this is actually the "correct" way to do this, since I've never done frontend professionally, but it's the way the rest of the code is, so let's just be consistent for now. If it's an issue, it'll get resolved later when the frontend gets a professional makeover.

Reply:
To answer your question above: I did look over the chessground-examples repo. That helped me understand how to “select” themes with the CSS class names. But I was basing most of my decision on the defaults in [chessground](https://github.com/lichess-org/chessground) and how [lichess’ main repo](https://github.com/lichess-org/lila/tree/master/public) is setup.

You're probably right about the best way to import `.css` files. I’m not totally sure why the chessground defaults have to be imported into `.tsx`. I thought I read it somewhere, but it might have gotten that impression from this mysterious message in the [chessground-example/assets/chessground.css](https://github.com/lichess-org/chessground-examples/blob/master/assets/chessground.css)
```
/*
 * Chessground base css properties.
 *
 * You need to include the css files in themes folder in order to have the
 * board and pieces displayed!
 */

.cg-wrap {
  width: 320px;
  height: 320px;
  position: relative;
  display: block;
}
...
```
I don’t know what `themes folder` this is referencing.

Reply:
I think in case of an organization/style conflict, we should go with the Chessground example repo - it seems much more tractable than the main lichess repo, which is a nightmare to navigate (except for professional frontend guys and people who actively work on it, I'm sure...).

Agreed re: "themes" folder - no idea what it's talking about, especially when the layout and code is so clear on its own...